iocage_lib/ioc_json.py

```diff
@@ -410,8 +410,6 @@ def json_load(self):
                                        f" Please destroy {uuid} and recreate"
                                        " it.")
 
-        conf = self.json_check_config(conf)
```

Comment:
Why the removal? This is important for anything calling json_load that the JSON is validated and properly updated with defaults.